CHANGELOG.md

```diff
@@ -4,6 +4,56 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-rc.4](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-rc.4) - 2023-01-30
+
+In this release we fixed bugs and inconsistencies between zcashd and zebrad in the output of the `getblocktemplate` RPC mehod. In addition, we added the proposal mode feature while we continue the effort of adding and testing mining pool RPC methods. 
+
+### Breaking Changes
+
+- Strings in zebra configuration file now use double quotes, caused by bumping the `toml` crate. Old configs will still work [#6029](https://github.com/ZcashFoundation/zebra/pull/6029)
+
+### Security
+
+- Verify the lock times of mempool transactions ([#6027](https://github.com/ZcashFoundation/zebra/pull/6027))
+- Avoid selecting duplicate transactions in block templates ([#6006](https://github.com/ZcashFoundation/zebra/pull/6006))
+- Calculate getblocktemplate RPC testnet min and max times correctly ([#5925](https://github.com/ZcashFoundation/zebra/pull/5925))
+- Fix Merkle root transaction order in getblocktemplate RPC method ([#5953](https://github.com/ZcashFoundation/zebra/pull/5953))
+- Bump bumpalo from 3.8.0 to 3.12.0 ([#6015](https://github.com/ZcashFoundation/zebra/pull/6015))
+- Bump libgit2-sys from 0.14.0+1.5.0 to 0.14.2+1.5.1 ([#6014](https://github.com/ZcashFoundation/zebra/pull/6014))
+- Bump tokio from 1.24.1 to 1.24.2 ([#5995](https://github.com/ZcashFoundation/zebra/pull/5995))
```

Comment:
It might help users if we explain what the security issue is, and what its impact is.
```suggestion
- Bump bumpalo from 3.8.0 to 3.12.0, removing undefined behaviour on `wasm` targets. These targets are not supported Zebra platforms. ([#6015](https://github.com/ZcashFoundation/zebra/pull/6015))
- Bump libgit2-sys from 0.14.0+1.5.0 to 0.14.2+1.5.1, to ensure that SSH server keys are checked. Zebra only uses `libgit2` during builds, and we don't make SSH connections. ([#6014](https://github.com/ZcashFoundation/zebra/pull/6014))
- Bump tokio from 1.24.1 to 1.24.2, to fix unsoundness. The unsound methods are not directly used by Zebra. ([#5995](https://github.com/ZcashFoundation/zebra/pull/5995))
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/6045/commits/6862ce4e3dd43e5dab6f87e2d49b0069897f2d60